doc/nrf/config_and_build/multi_image.rst

```diff
@@ -203,7 +203,7 @@ Variables in the parent image are not propagated to the child image, with the fo
 
 * Any variable named ``<IMAGE_NAME>_VARIABLEONE`` in a parent image is propagated to the child image named ``<IMAGE_NAME>`` as ``VARIABLEONE``.
   See `Temporary variables in child images`_ for more information.
-* CMake build settings, such as ``BOARD_DIR``, build type, toolchain info, partition manager info, and similar are always passed to child images.
+* CMake build settings, such as ``BOARD_DIR``, suffixed configuration, build type, toolchain info, partition manager info, and similar are always passed to child images.
```

Comment:
Is the `suffixed configuration` passed to child images already?

Reply:
I would consider to explicitly mention that using `FILE_SUFFIX` is not yet recommended for child images and we should stick to `CONF_FILE` approach (similar for Partition Manager static configuration).

I would also explicitly state that `FILE_SUFFIX` is not automatically propagated to child images. Unfortunately, it does not seem to work even if I pass information manually - e.g. as `-Db0_FILE_SUFFIX=release` (tested with nRF Desktop using the following command: `west build -b nrf52840dk_nrf52840 -p -d suffix -- -DFILE_SUFFIX=release -Db0_FILE_SUFFIX=release`)

Reply:
Removed references to child images for FILE_SUFFIX.

Reply:
@nordicjm, is it expected to work already? Maybe something is still missing in the implementation